visualization/app/codeCharta/ui/ribbonBar/ribbonBar.component.scss

```diff
@@ -127,7 +127,8 @@ ribbon-bar-component {
 		margin: 0;
 	}
 
-	md-card.custom-configs-card {
+	md-card.custom-configs-card,
+	md-card.ai-card {
 		max-width: 120px;
```

Comment:
(Ich weiß das du diese Stelle nicht angefasst hast) Max-width und Min-width auf die selbe breite zu stellen ist effektiv einfach nur width:, damit könnte man sich eine Zeile Code und etwas verwirrung sparen. Das wäre nur relevant falls diese Container Flexboxen sind, dann ist mein Kommentar hinfällig.

Reply:
You are right. In this case `max-width` overwrites the settings in a parent class and I `min-width` should be at least `120px` for small screens.